openlibrary/utils/solr.py

```diff
@@ -47,7 +47,14 @@ def get(
         logger.info(f"solr /get: {key}, {fields}")
         resp = self.session.get(
             f"{self.base_url}/get",
-            params={'id': key, **({'fl': ','.join(fields)} if fields else {})},
+            params={
+                'id': key,
+                **(
+                    {'fl': ','.join([field for field in fields if field])}
```

Comment:
Let's check if we can why `None` is getting in here, but if it looks unclear after like 5min investigating, this is ok.